test/backend/routes.queries.spec.js

```diff
@@ -36,6 +36,7 @@ describe('Routes:', () => {
             fid: validFid,
             uids: validUids.slice(0, 2), // since this particular query only has 2 columns
             refreshInterval: 60, // every minute
+            cronInterval: null,
```

Comment:
These tests should pass without setting `cronInterval` to `null` (to reproduce what would happen if new Falcon runs an old `~/.plotly/connector/queries.yaml`).